Code/Editor/EditorFramework/EditorApp/EditorApp.moc.h

```diff
@@ -269,7 +273,9 @@ private Q_SLOTS:
   QApplication* s_pQtApplication;
   ezLongOpControllerManager m_LongOpControllerManager;
   ezEditorEngineProcessConnection* s_pEngineViewProcess;
-  QTimer* m_pTimer;
+  QTimer* m_pTimer;
```

Comment:
We should really just initialize the variables here. The existing code ist just old and therefore doesn't use that feature yet.